setonix/repo_setonix/packages/plumed/package.py

```diff
@@ -91,7 +91,7 @@ class Plumed(AutotoolsPackage):
 
     parallel = False
 
-    def apply_patch(self, other):
+    def apply_patch(self, other, force=False):
```

Comment:
Any reason for the force=False?

Reply:
Yep, this is to cover the default case (where we probably do want to keep the plumed patch check) as it applies in other recipes, such as the one for gromacs.